File: 0977-squares-of-a-sorted-array/0977-squares-of-a-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var sortedSquares = function(nums) {
    let ans=[];
    for(let i=0; i<nums.length; i++){
        let val=nums[i]*nums[i];
        ans.push(val);
    }
    ans.sort((a, b) => a - b);
    return ans;
};

File: 0977-squares-of-a-sorted-array/0977-squares-of-a-sorted-array.cpp
class Solution {
public:
    vector<int> sortedSquares(vector<int>& nums) {
        vector<int>ans;
        int n=nums.size();
        for(int i=0; i<n; i++){
            int val=nums[i]*nums[i];
            ans.push_back(val);
        }
        sort(ans.begin(), ans.end());
        return ans;
    }
};